features/createroom/impl/src/main/kotlin/io/element/android/features/createroom/impl/CreateRoomFlowNode.kt

```diff
@@ -74,7 +75,12 @@ class CreateRoomFlowNode @AssistedInject constructor(
                 createNode<CreateRoomRootNode>(context = buildContext, plugins = listOf(callback))
             }
             NavTarget.NewRoom -> {
-                createNode<ConfigureRoomFlowNode>(context = buildContext, plugins = emptyList())
+                val callback = object : ConfigureRoomNode.Callback {
+                    override fun onRoomCreated(roomId: RoomId) {
+                        plugins<CreateRoomEntryPoint.Callback>().forEach { it.onOpenRoom(roomId) }
```

Comment:
Maybe rename to `onRoomCreated` (it has more sense to me regarding the module contract)

```suggestion
                        plugins<CreateRoomEntryPoint.Callback>().forEach { it.onRoomCreated(roomId) }
```

Or maybe the module can return an existing room sometimes (ex: dm already exist)?

In this case, maybe rename to `onSuccess` or `onDone`.

Reply:
https://github.com/vector-im/element-x-android/pull/324/commits/2403775857c8b791a64a4e107a25c5bbd9cd6abc